pages/common/dvc-diff.md

```diff
@@ -0,0 +1,24 @@
+# dvc diff
+
+> Show changes in DVC tracked file and directories added/deleted/modified.
+> More information: <https://dvc.org/doc/command-reference/diff>.
+
+- Compare DVC tracked file from different Git commit/tag/branch w.r.t current workspace:
```

Comment:
Perhaps this will read better?
```suggestion
- Compare DVC tracked files from different Git commits, tags, and branches w.r.t the current workspace:
```